djangorestapi/api/tests.py

```diff
@@ -1,3 +1,17 @@
 from django.test import TestCase
```

Comment:
Add a doc string at the top of the file

Reply:
Okay will check on that.

Reply:
Use the Andela best practices for naming the PRs